### ISSUE TITLE
`get_weigh` returns the transposed results for a SimpleWeighedDiGraph 

### ISSUE THREAD

**Issue by jrklasen** [JuliaGraphs/SimpleWeightedGraphs.jl#39]
See the following example for SimpleWeightedGraphs v1.3.0:

```
using SimpleWeightedGraphs
using Graphs

g = SimpleWeightedDiGraph(3)
add_edge!(g, 1, 2, 0.5)
get_weight(g, 1, 2) # gives 0.0 instead of 0.5
get_weight(g, 2, 1) # gives 0.5 instead of 0.0
```

**Comment by simonschoelly** [comment]
That sounds rather serious. I can indeed reproduce the error with `v1.3.0`. I also saw, that we did not have the error with version `v1.2.1`.

**Comment by simonschoelly** [comment]
I suspect the issue was introduced in #14.

We probably just have to change
```julia
get_weight(g::AbstractSimpleWeightedGraph, u::Integer, v::Integer) = weights(g)[v, u]
```
into
```julia
get_weight(g::AbstractSimpleWeightedGraph, u::Integer, v::Integer) = weights(g)[u, v]
```
and write some tests to avoid such issues in the future. PRs are welcome :)

**Comment by gdalle** [comment]
Good catch! I'll take care of it as soon as #38 is merged

**Comment by jrklasen** [comment]
I'm not familiar with the code base, but be aware that is not happening for other graph types of this package. If they all belong to the same abstract type, this fix wouldn't work.

**Comment by simonschoelly** [comment]
The only other concrete graph type in this package is the undirected version, so we won't have an issue there.

**Comment by gdalle** [comment]
Solved by #40
